wdio.conf.js

```diff
@@ -1,9 +1,12 @@
 const path = require('path');
 // eslint-disable-next-line import/no-extraneous-dependencies
-const { config } = require('@cerner/terra-functional-testing/lib/config/wdio.conf');
+const defaultWdioConfig = require('@cerner/terra-functional-testing/lib/config/wdio.conf');
```

Comment:
What is the reason for the changes in this file?

Reply:
It was decided to make the terra-pills a scoped package in the previous PR. So changes was for wdio's to run without errors.

[Here](https://github.com/cerner/terra-framework/pull/1442#discussion_r687233829) is the link to the comment

Should we make this a scoped package? @neilpfeiffer  thoughts?

Reply:
I have kept it consistent with terra-core, do you want me to de-structure to `{config}`? LMK.

Reply:
I didn't know terra-core did it this way. How it was on this file should be the proper/cleaner way. I don't have a strong preference either way so we can just leave it the way you have it.